CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 1.8.0 (October 18, 2022). Tested on Artifactory 7.46.6 and Xray 3.59.4
+
+IMPROVEMENTS:
+
+* resource/xray_watch: add functionality to apply `path_ant_filter` for the watched repositories. [#PR]()
```

Comment:
Missing PR and Issue links

Reply:
There is no issue in GitHub for this change. PR link will be committed next.